ccm_web/client/src/App.tsx

```diff
@@ -1,18 +1,61 @@
 import React from 'react'
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import { BrowserRouter as Router, Link as RouterLink, Route, Switch } from 'react-router-dom'
+import { Breadcrumbs, Link, makeStyles, Typography } from '@material-ui/core'
 
 import ConsumerTest from './components/ConsumerTest'
-import Home, { mergeSectionProps } from './pages/Home'
-import MergeSections from './pages/MergeSections'
+import Home, { featureCardProps } from './pages/Home'
 import './App.css'
 
+const useStyles = makeStyles((theme) => ({
+  breadcrumbs: {
+    paddingLeft: 25,
+    paddingTop: 25
+  }
+}))
+
 function App (): JSX.Element {
+  const classes = useStyles()
   return (
     <div className='App'>
       <Router>
+        <div className={classes.breadcrumbs}>
+          <Route>
+              {({ location }) => {
+                const pathnames = location.pathname.split('/').filter(x => x)
```

Comment:
I think this is cool and extensible, but something about it feels a bit fragile to me (all the string manipulation and matching?) or overly complex. What happens if there is a trailing slash? (legitimate question, I haven't thought it through yet.)

Here's perhaps one alternative. Each component in `pages` uses a `Layout` component that includes a `BreadCrumbs` component and a footer of some sort, as well as allowing children. You can pass hard-coded crumb information (that you know because you're in a component) into `Layout`. One positive side effect of this would be that we minimize code the amount of code in `App`.

Food for thought. Happy to discuss the pros and cons with you if you'd like.

Reply:
I'd like to see how this plays out. It's based on how the  material docs demonstrated interacting with react-router.

Reply:
Okay, can you link to the documentation you're talking about? Danke

Reply:
Mmm, here I guess: https://material-ui.com/components/breadcrumbs/#integration-with-react-router

I'll mark this as resolved.